src/app/zap-templates/zcl/data-model/chip/operational-state-dishwasher-cluster.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="OperationalStateEnum" type="ENUM8">
```

Comment:
Shouldn't we just add this cluster id to the OperationalStateEnum in the main Operational State cluster?

Reply:
if I remove this OperationalStateEnum in operational-state-dishwasher-cluster.xml, it will build err. just like commit: 0618092fff552675f4d179e5fabaf85e1ded6fb8,  do you have any suggestion?

Reply:
I'm not following.  What that commit shows is this cluster no longer using the generic OperationalStateStruct and OperationalStateEnum.  But why is it not using that?  What is the actual build error you get, and using what code @mideayanghui ?